### PR TITLE
Update subgraph setup to new style and fix a few issues

### PR DESCRIPTION
**Pull request by 0xGabi**
Use the same style that Ale introduce on the app universal subgraph and fixed a few issues detected after an in-depth review of the data fetched from the blockchain.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,8 +3,15 @@
   "private": true,
   "devDependencies": {},
   "workspaces": {
-    "packages": ["packages/*"],
-    "nohoist": ["packages/*-subgraph"]
+    "packages": [
+      "packages/*"
+    ],
+    "nohoist": [
+      "**/@graphprotocol/graph-ts",
```
Comment (kanaagawa) [inline package.json:10]: We were using the nohoist wrong. We needed to select the specific node_module dependencies.

File: .gitignore [hunk 2]
```diff
@@ -6,7 +6,7 @@ dist/
 packages/dao-subgraph/src/helpers/ens.ts
 
 # Production subgraph
-types
+generated
```
Comment (kanaagawa) [inline .gitignore:9]: @ajsantander should we remove the generated file on the voting and token-manager specific subgraphs?

Comment (kanaagawa) [reply .gitignore:9]: I remove it

**Comment by 0xGabi** [comment]
The subgraphs were deployed to the official Aragon account

**Pull request merged**